package.json

```diff
@@ -4,8 +4,8 @@
   "description": "DXswap Decentralized Application",
   "license": "GPL-3.0-or-later",
   "scripts": {
-    "start": "yarn && yarn codegen:socket && react-scripts start",
```

Comment:
Should we do it on pre-install as before? I didn't noticed this was removed and only ran `yarn codegen` after running `yarn install` and `yarn start` a couple of times

Reply:
Since I added `yarn & react-scripts start`  you dont want to do `yarn install`. 

`yarn = yarn install`

I felt dont want to run it always 😞